docs/source/notebooks/finch-usage.ipynb

```diff
@@ -140,77 +144,991 @@
      "output_type": "stream",
      "text": [
       "Process status:  ProcessSucceeded\n",
-      "Link to process output:  https://pavics.ouranos.ca/wpsoutputs/6098a61c-7ae6-11eb-9131-9c305b4af68f/frost-days_SRES-A2-experiment_20460101-20650101.nc\n"
+      "Link to process output:  http://localhost:5000/outputs/62b3ef56-859a-11eb-a5eb-a44cc85b43bf/frost-days_SRES-A2-experiment_20460101-20650101.nc\n"
```

Comment:
Did you run `refresh-notebooks`?  `refresh-notebooks` should automatically replace `localhost:5000` with `pavics.ouranos.ca`

Reply:
Ok, but it also seems to strip the html output. After refresh, I'm back with text output... weird.